baselines/constraint/constraint_wrapper.py

```diff
@@ -41,6 +41,27 @@ def __init__(self,
             self.logs = None
 
     def augment_obs(self, ob):
+        """Augment the current observation.
+
+        Parameters
+        ----------
+        self: ConstraintEnv
+            A data structure that serves as 
+            a wrapper to the Gym environment.
+
+        ob: Tensor?
```

Comment:
In Atari this is going to be a LazyFrames object, but it's ultimately a numpy array, not a Tensor